### PR TITLE
Latest misc LFS changes

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: options/rtdl/generic/linker.cpp [hunk 1]
```diff
@@ -22,6 +22,12 @@ constexpr bool logBaseAddresses = false;
 constexpr bool logRpath = false;
 constexpr bool eagerBinding = true;
 
+namespace {
+
+bool rpathWarned = false;
```
Comment (Geertiebear) [inline options/rtdl/generic/linker.cpp:27]: Is this change really necessary?

**Comment by Geertiebear** [comment]
What led you to making the ``_save_pos()`` change? It LGTM I'm just curious.

File: options/rtdl/generic/linker.cpp (hunk 1 above)
Comment (no92) [reply options/rtdl/generic/linker.cpp:27]: It massively spams the log when running some applications, so warning once seemed sufficient.

**Comment by no92** [comment]
I don't recall the details as it's been a few weeks but IIRC it was seeking on sysfs or proc files in managarm.

File: options/rtdl/generic/linker.cpp (hunk 1 above)
Comment (Geertiebear) [reply options/rtdl/generic/linker.cpp:27]: Alright makes sense. I would prefer if it wasn't a global variable though, putting it as a static variable in the case scope should work. Also put a comment explaining that this is both temporary and to reduce log-spam for some applications.

**Review by Geertiebear — APPROVED** [review]
LGTM
